spec/features/staff/users_can_sign_out_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.feature "Users can sign out" do
+  scenario "success" do
+    ClimateControl.modify BULLET_DEBUG: "false" do
```

Comment:
Out of interest, what's this for?

Reply:
All I can tell you is it turns off the Bullet gem's debug output, which as far as I can tell we aren't turning on explicitly. If I remove it, the spec still passes. @CristinaRO any clues?

Reply:
Ah ok. I think we can lose it then.

Reply:
[This commit](a5a016a2567a0ec280b5a2e48dd8a3798cd983df) has a bit more detail but I think it applies only to the `AUTH0` vars.